web/repos.jspf

```diff
@@ -263,8 +293,37 @@ Copyright (c) 2015, 2016, Oracle and/or its affiliates. All rights reserved.
                                 if (branch == null) {
                                     branch = "N/A";
                                 }
+                                String currentVersion = ri.getCurrentVersion();
+                                if (currentVersion == null) {
+                                    currentVersion = "N/A";
+                                }
                                 %><td><%= Util.htmlize(type) %></td><%
                                 %><td><%= Util.htmlize(parent) %> (<%= Util.htmlize(branch) %>)</td><%
+                                %><td><%
```

Comment:
Now it looks like this:
![screenshot from 2016-11-04 12-24-41](https://cloud.githubusercontent.com/assets/6997160/20003978/c29d49ba-a289-11e6-9f1f-de6684e881ca.png)

Which has not really necessary margin, can you avoid this?